crates/bevy_ui/src/geometry.rs

```diff
@@ -22,7 +22,7 @@ use std::ops::{Div, DivAssign, Mul, MulAssign};
 /// ```
 ///
 /// If you define opposite sides of the position, the size of the UI element will automatically be calculated
-/// if not explicitly specified. This means that if you have a [`Size`] that uses [`Val::Undefined`](crate::Val::Undefined)
+/// if not explicitly specified. This means that if you have a [`Size`] that uses [`Val::Auto`](crate::Val::Auto)
```

Comment:
The comment is for the "title":

> A type which is commonly used to define **positions**, margins, paddings and borders.

`position` should be removed here.

Actually, if this was used for position, when used with `PositionType::Absolute` it'll probably end up in surprising behavior given that it defaults to `0px` (I think it would cover the whole parent relative node… at least in CSS would).

Reply:
Yes, meant to delete this.

You point about positions is correct, the default for UiRect's fields is changed from `Val::Undefined` to `Val::Px(0.)` which is why it can no longer be used to represent positions.